doc/monitoring/api_reference.rst

```diff
@@ -343,10 +343,42 @@ Metrics functions
     some global label, the method argument value will be used.
 
     Note that both labels names and values in label_pairs are treated as strings.
+
 ..  function:: collect()
 
     Collect observations from each collector.
 
+..  class:: registry
+
+    ..  method:: unregister(collector)
+
+        Removes collector from registry
+
+        :param table collector: collector that has to be removed
+
+        Example:
+
+        ..  code-block:: lua
+            local collector = metrics.gauge('some-gauge')
+
+            -- after some time we don't need it anymore
+
+            metrics.registry:unregister(collector)
+
+    ..  method:: find(kind, name)
```

Comment:
Possible return values? It is obvious that in case of success it returns a collector, but what in case of an error?